pkg/agent/datapath/localBridge.go

```diff
@@ -189,7 +189,12 @@ func (l *LocalBridge) cleanLocalIPAddressCache(timeout int) {
 }
 
 func (l *LocalBridge) processLocalEndpointUpdate(arpIn protocol.ARP, inPort uint32) {
-	if !l.isOfPortExists(inPort) {
+	endpoint, isExist := l.getEndpointByPort(inPort)
+	if !isExist {
+		return
+	}
+
+	if endpoint.MacAddrStr != arpIn.HWSrc.String() && endpoint.IPAddr != nil {
```

Comment:
This is incorrect, the endpoint.IPAddr is not the endpoint local IP address, you need to check whether there is local IP for this endpoint. 
Record the endpoint local ip if it has, also need to update the endpoint local ip if the ip address flushed.